code/modules/mob/living/simple_animal/friendly/adhomai.dm

```diff
@@ -283,11 +283,6 @@
 	meat_amount = 4
 	faction = "Adhomai"
 	vehicle_version = /obj/vehicle/animal/climber
-	natural_armor = list(
-		melee = ARMOR_MELEE_MEDIUM,
-		bullet = ARMOR_BALLISTIC_MINOR,
-		bomb = ARMOR_BOMB_MINOR
-	)
```

Comment:
I am not sure removing the armor is a proper solution for the runtime

Reply:
Removed this as the issue was solved properly by #18856